src/banner/_base.scss

```diff
@@ -27,11 +27,17 @@ header {
 			}
 		}
 
-		object,
-		img {
+		// Needed for backwards compatibility with GCWeb 4.0.27 and below's HTML markup.
+		// Remove object references from all of this theme's logo selectors in GCWeb 4.1+.
+		img,
```

Comment:
Minor, if you keep the order of tags the diff will be minimized

Reply:
@nschonni I had thought about that while prepping this and its sister PRs. But since I was already inserting a comment nearby that downplayed the use of the ``object`` element, I felt like it'd also be worthwhile to take the opportunity to place it below ``img`` in the selector to further de-emphasize it. Plus it's more alphabetical this way :).